neuro_san/test/driver/data_driven_agent_test_driver.py

```diff
@@ -168,8 +166,10 @@ def parse_hocon_test_case(self, hocon_file: str) -> Dict[str, Any]:
 
         :param hocon_file: The name of the hocon from the fixtures directory.
         """
-        test_path: str = self.fixtures.get_file_in_basis(hocon_file)
-        hocon = EasyHoconPersistence()
+        test_path: str = hocon_file
+        if self.fixtures is not None:
+            test_path = self.fixtures.get_file_in_basis(hocon_file)
+        hocon = EasyHoconPersistence(must_exist=True)
```

Comment:
Only use the fixtures' basis  if we were sent one in the constructor.
Also: If the test case hocon file is not found, assert here and now.